internal/throttler/throttler.go

```diff
@@ -16,6 +16,10 @@ type Throttler struct {
 	bytesPerSecond float64
 }
 
+func NewThrottlerBytesPerSecond(bytesPerSec float64) *Throttler {
```

Comment:
Should this be:
```suggestion
func NewThrottlerBytesPerSecond(bytesPerSec BytesPerSec) *Throttler {
```
for clarity?

Reply:
Done